content/docs/user-guide/components/reference/terrain/world.md

```diff
@@ -21,10 +21,19 @@ You must add this component to the *Level* entity, the parent of all entities in
 
 | Property | Description | Values | Default |
 | - | - | - | - |
-| World Bounds (min) | The minimum value for the worlds bounds. The **min** value must be less than the **max** value.| Vector3: -2048.0 to 2048.0 | X:`0.0` Y:`0.0` Z:`0.0` |
-| World Bounds (max) | The maximum value for the worlds bounds. The **max** value must be greater than the **min** value. | Vector3: -2048.0 to 2048.0 | X:`1024.0` Y:`1024.0` Z:`1024.0` |
-| Height Query Resolution (m) | The resolution required between each height sample, in meters. | Vector2: 0.001 to Infinity | X:`1.0` Y:`1.0` |
+| Min Height | The minimum value for the height of terrain. The **min** value must be less than the **max** value.| -65536.0 to 65536.0 | 0.0 |
+| Max Height | The maximum value for the worlds bounds. The **max** value must be greater than the **min** value. | -65536.0 to 65536.0 | 1024.0 |
+| Height Query Resolution (m) | The distance between each height sample position, in meters. | 0.1 to Infinity | 1.0 |
+| Surface Data Query Resolution (m) | The distance between each surface data sample position, in meters. | 0.1 to Infinity | 1.0 |
 
-{{< note >}}
-Because large numbers of terrain height samples can take a long time to calculate, O3DE will not allow the number of samples to exceed 8 million.
-{{< /note >}}
+
+## Usage 
+
+The **Min Height** and **Max Height** bounds should be set to as small a range as possible for the most accurate height values. For instance, the terrain renderer uses 16-bit height values, which provides 65536 possible distinct heights. If the difference between the min and max height is 1 km, that's a resolution of about 1.5 cm.
+
+**Height Query Resolution** and **Surface Data Query Resolution** are synchronization points for terrain height and surface data used by different terrain systems like physics and rendering. They effectively create grids across the world - one for height, and one for surface data. All height and surface data providers will be queried at this grid resolution regardless of the resolution of the data in providers. Ideally, source data in providers with quantized data (like image gradients) will match the query resolution used.
+
+For example, if a heightmap with 10 texels/m is used with a terrain that has a 1m height query resolution, 9/10 of the texels will be ignored. If the heightmap has 1 texel/m with a 1 m height query resolution, then all of the texels will be used.
```

Comment:
```suggestion
For example, if a terrain uses a heightmap with 10 texels/meter with a 1 meter height query resolution, then 9 out of 10 of the texels are ignored. If the heightmap has 1 texel/meter with a 1 meter height query resolution, then all of the texels are used.
```

Reply:
After reading this again I'm going to change "heightmap" to "heightmap image" to be clearer since we're talking about an image source that happens to contain heightmap information.